Sources/momento/messages/responses/cache/data/list/CacheListFetch.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+public protocol CacheListFetchResponse {}
+
+public class CacheListFetchHit: CacheListFetchResponse {
+    public let valueListString: [String]
+    public let valueListData: [Data]
+    
+    init(values: [Data]) {
+        self.valueListData = values
+        self.valueListString = values.map { String(decoding: $0, as: UTF8.self) }
```

Comment:
We should think about making these [lazy properties](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/properties/#Lazy-Stored-Properties), but we can handle that as a separate effort if we decide it's worth it. Just putting it on your radar.